### ISSUE TITLE
Support for UI.com account with MFA?

### ISSUE THREAD

**Issue by rsnodgrass** [robinsmidsrod/unifi-controller-cli#1]
Does this support accounts which are setup with multi-factor authentication (MFA or 2FA)?  I can't seem to get this to connect to the controller, but if I click on the link emitted in the logs it does load in the browser and shows json data.  My guess is that this does not take the 6-digit multi-factor authentication token?

**Comment by robinsmidsrod** [comment]
Indeed, it does not support MFA. Do you have any information on how to authenticate API requests with MFA enabled?

**Comment by rsnodgrass** [comment]
No, but I’ll do some research!!
  
  
  
    
  

  
  
>

**Comment by rsnodgrass** [comment]
What I've found is that the recommendation is to create another "Local User" on the controller that does not have MFA activated.  This account is used explicitly for API access.

**Comment by robinsmidsrod** [comment]
Yup, this is what I seem to recall as well, when I did some research about it way-back-when. I guess that means we can close this issue?

**Comment by rsnodgrass** [comment]
I think so.  Now, I can't get Net::SSLeay to actually install on the Mac that I use all the time. :(
